vignettes/stan-help.Rmd

```diff
@@ -0,0 +1,423 @@
+---
+title: Resources for help with Stan
+description: "How to address issues you may encounter with Stan"
+author: Michael DeWitt
+output: rmarkdown::html_document
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+vignette: >
+  %\VignetteIndexEntry{Help with Stan}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+
+<!-->
+
+- [ ] What is Stan
```

Comment:
To remove?

Reply:
Removing comments and placing [here](https://gist.github.com/medewitt/caf2af7095b2d851a0d9de736b489987) until I get these next iteration points into a new issue